lib/manager/git-submodules/extract.ts

```diff
@@ -106,7 +106,8 @@ export default async function extractPackageFile(
         try {
           const [currentValue] = (await git.subModule(['status', path]))
             .trim()
-            .split(/[+\s]/);
+            .replace(/[-+]/, '')
```

Comment:
```suggestion
            .replace(/^[-+]/, '')
```

Not sure about this, but couldn't a `-` be in the name of a submodule otherwise?

Reply:
> I wasn't able update a submodule using `update-index` and put the change in 'Changes not staged for commit'. I'm open to suggestions here.

Ohh I wasn't aware they need to be unstaged, in that case I also don't know of way without a checkout :/